src/utils.ts

```diff
@@ -327,6 +327,10 @@ function patchMatrixClientForConciseExceptions() {
                     enumerable: false,
                 });
             }
+            if (!LogService.level.includes(LogLevel.TRACE)) {
+                // Remove stack trace to reduce impact on logs.
+                error.stack = "";
+            }
             return cb(error, response, resBody);
```

Comment:
Who calls `LogService.error` after this though?

Reply:
matrix-bot-sdk, in http.js

Reply:
OK, is there a reason why they do this https://github.com/turt2live/matrix-bot-sdk/blob/e7a77fd126801694b723b79f16fb6031e16b4d1b/src/http.ts#L30 instead of including that information in the error object? Could we add that in our wrapper? Then we would just be able to silence `MatrixHttpClient` completely.

Reply:
It'd also mean we don't have to run logs in debug to be able to report useful errors

Reply:
I'd love to. I haven't been able to get any review on my PRs for matrix-bot-sdk, but perhaps if we ping hard enough in the relevant room, we could do that, too :)

Reply:
We wouldn't have to modify matrix-bot-sdk to do any of this though.

Reply:
I still think we need to do this and did not explicitly approve the PR for this reason, but ok.